[2022-09-05T18:11:06.842+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: our_dag_with_python_operator_06.get_name manual__2022-09-05T18:11:06.077996+00:00 [queued]>
[2022-09-05T18:11:06.861+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: our_dag_with_python_operator_06.get_name manual__2022-09-05T18:11:06.077996+00:00 [queued]>
[2022-09-05T18:11:06.862+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-05T18:11:06.862+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-09-05T18:11:06.862+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-05T18:11:06.885+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): get_name> on 2022-09-05 18:11:06.077996+00:00
[2022-09-05T18:11:06.892+0000] {standard_task_runner.py:52} INFO - Started process 3920 to run task
[2022-09-05T18:11:06.896+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_06', 'get_name', 'manual__2022-09-05T18:11:06.077996+00:00', '--job-id', '334', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpan60w_18', '--error-file', '/tmp/tmpyvzr_15e']
[2022-09-05T18:11:06.897+0000] {standard_task_runner.py:80} INFO - Job 334: Subtask get_name
[2022-09-05T18:11:06.997+0000] {task_command.py:371} INFO - Running <TaskInstance: our_dag_with_python_operator_06.get_name manual__2022-09-05T18:11:06.077996+00:00 [running]> on host c033faf33c57
[2022-09-05T18:11:07.105+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ancordss
AIRFLOW_CTX_DAG_ID=our_dag_with_python_operator_06
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-09-05T18:11:06.077996+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-05T18:11:06.077996+00:00
[2022-09-05T18:11:07.143+0000] {python.py:173} INFO - Done. Returned value was: None
[2022-09-05T18:11:07.170+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_06, task_id=get_name, execution_date=20220905T181106, start_date=20220905T181106, end_date=20220905T181107
[2022-09-05T18:11:07.235+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-05T18:11:07.289+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
